### DISCUSSION TITLE
Packaging an ARC

### DISCUSSION THREAD

**Discussion by dvs-frbg** [nfdi4plants/ARC-specfication#10] [Comments and Suggestions]
Publishing a digital object at some repository usually requires a form of packaging (simply to stuff a full blown directory structure with (many) files in it into a single file which is easier to handle from the repo perspective; of course that induces some forms of unpackaging either through a user or a tool/workflow when it is getting reused). For digital archives formats like BagIt, OCFL are suggested, compressed "archives" (ZIP, TGZ, RAR, whatever) are common as well. Optimally, it follows some (widely used) standard and avoids proprietary packaging algorithms. Such information could be added to the "Shareable and Publishable ARCs" section/notes.

**Comment by muehlhaus** [comment]
As this is an export of an ARC , the specification suggests that the target repro specifies the requirements. 
I would suggested to place this idea into the Invenio ARC submission guidelines for ARCs.

**Comment by chgarth** [comment]
I would see general "archiving" of ARCs as "exports". Exports can be, but need not be, bidirectional.

To anticipate the need to package an ARC into a single file, the "git bundle" export was included in the spec, which is a well-defined format (if not used very much). This is guaranteed to be bidirectional, so defines at least one single-file package that we can definitely rely on.
